### DISCUSSION TITLE
Support `WIX_IS_NETFRAMEWORK_481_OR_LATER_INSTALLED`

### DISCUSSION THREAD

**Discussion by ww898** [wixtoolset/issues#7426] [Questions]
Based on https://github.com/wixtoolset/issues/issues/7425: This is not discussion. Nothing to be discussed here. This is real feature request. .NET Framework 4.8.1 is here: https://dotnet.microsoft.com/en-us/download/dotnet-framework/net481

**Comment by robmen** [comment]
#7425 is not a feature request. It is a single sentence that insinuates we owe you something even though, AFAIK, we've never met you before. 

If you want to open a feature request, please complete the feature request template fully. Since you've never contributed to this community, I highly suggest you do so respectfully. The volunteers on this OSS project are very likely to ignore you if you are rude and/or act entitled. If you want to make demands, [become a customer](https://www.firegiant.com/services/support-programs/).

Note: please do not go open a feature request for .NET Framework 4.8.1 support. That was already added to WiX v4.
